game/screen_intro.cc

```diff
@@ -181,15 +181,15 @@ void ScreenIntro::populateMenu() {
 			// Process items that belong to that submenu
 			for (auto const& item: submenu.items) {
 				ConfigItem& c = config[item];
-				opts.push_back(MenuOption(_(c.getShortDesc()), _(c.getLongDesc())).changer(c));
+				opts.push_back(MenuOption(c.getShortDesc(), c.getLongDesc()).changer(c));
 			}
-			configmain.push_back(MenuOption(_(submenu.shortDesc), _(submenu.longDesc), imgConfig).submenu(opts));
+			configmain.push_back(MenuOption(submenu.shortDesc, submenu.longDesc, imgConfig).submenu(opts));
 		} else {
-			configmain.push_back(MenuOption(_(submenu.shortDesc), _(submenu.longDesc), imgConfig).screen(submenu.name));
+			configmain.push_back(MenuOption(submenu.shortDesc, submenu.longDesc, imgConfig).screen(submenu.name));
 		}
 	}
-	m_menu.add(MenuOption(_("Configure"), _("Configure audio and game options."), imgConfig).submenu(configmain));
-	m_menu.add(MenuOption(_("Quit"), _("Leave the game."), imgQuit).screen(""));
+	m_menu.add(MenuOption(translate_noop("Configure"), translate_noop("Configure audio and game options."), imgConfig).submenu(configmain));
+	m_menu.add(MenuOption(translate_noop("Quit"), translate_noop("Leave the game."), imgQuit).screen(""));
```

Comment:
Same in this file. Please do not use `translate_noop`. Use `_("dsdfdf")` instead

Reply:
same, I don't want the menu to be created with the translated string. I need to have all internal code use english and just render it with translations (when calling opt.getValue())